Spigot-Server-Patches/0167-Anti-Xray.patch

```diff
@@ -0,0 +1,997 @@
+From 00c7f062c72bbed69578cddf0f21e236afaadb0c Mon Sep 17 00:00:00 2001
+From: BuildTools <unconfigured@null.spigotmc.org>
+Date: Mon, 27 Jun 2016 22:47:40 +0200
+Subject: [PATCH] Anti-Xray
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index e3fe8f4..fbcc680 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -1,7 +1,9 @@
+ package com.destroystokyo.paper;
+ 
++import java.util.Arrays;
+ import java.util.List;
+ 
++import com.destroystokyo.paper.antixray.AntiXray;
+ import org.bukkit.Bukkit;
+ import org.bukkit.configuration.file.YamlConfiguration;
+ import org.spigotmc.SpigotWorldConfig;
+@@ -382,4 +384,34 @@ public class PaperWorldConfig {
+     private void elytraHitWallDamage() {
+         elytraHitWallDamage = getBoolean("elytra-hit-wall-damage", true);
+     }
++
++    public boolean antiXray;
++    public int engineMode;
++    public List<Object> hiddenBlocks;
++    public List<Object> replaceBlocks;
++    public int maxChunkY;
++    public boolean asynchronous;
++    public int neighborsMode;
++    public AntiXray antiXrayInstance;
++    private void antiXray() {
++        antiXray = getBoolean("anti-xray.enabled", false);
```

Comment:
If anti-xray is not enabled, can the rest of the code in this method be skipped?